use-lazy-query.test.ts

```diff
@@ -69,7 +65,6 @@ describe("useLazyQuery", () => {
         expect(renderHookResult.result.current[1].error).toBe(null);
         expect(renderHookResult.result.current[1].loading).toBe(false);
         expect(renderHookResult.result.current[1].data).toBe(null);
-        expect(renderHookResult.result.all.length).toBe(1);
```

Comment:
Does `result.all` no longer exist?

Reply:
unfortunately not

https://github.com/testing-library/react-testing-library/pull/991

Reply:
I can potentially revert this PR, but I needed this in place to test the strict mode issues I was having - I couldn't find a way to run a unit test that showed the issue under React 17 (or at least was having a lot of trouble debugging due to the silenced console).